specs/Toast.md

```diff
@@ -0,0 +1,157 @@
+
+
+| Attribute | Notes |
+| --------- | ----------- |
+| [`role="status"`](https://www.w3.org/TR/wai-aria-1.1/#status) | A type of [live region](https://www.w3.org/TR/wai-aria-1.1/#dfn-live-region). Has implicit `aria-live="polite"` and `aria-atomic="true"` |
+| [`role="alert"`](https://www.w3.org/TR/wai-aria-1.1/#alert) | A type of [live region](https://www.w3.org/TR/wai-aria-1.1/#dfn-live-region). Has implicit `aria-live="assertive"` and `aria-atomic="true"` |
+| `aria-live="polite"` | This is generally the level we want to use for toasts. This is not needed when using `role="status"`. |
+| `aria-live="assertive"` | This steals the attention and should be avoided unless for critical messages. This is not needed when using `role="alert"` |
+| [`aria-atomic="true"`](https://www.w3.org/TR/wai-aria-1.1/#aria-atomic) | Always present all of the region, as opposed to just the parts that changed. |
+| [`role="region"`](https://www.w3.org/TR/wai-aria-1.1/#region) | This probably is not necessary for anything. Maybe the component that contains all the toasts. |
+
+
+API Specifications
+------------------
+
+### Props
+
+#### `<Toast>`
+
+| Name | Type | Default | Description |
+| ---- | ---- | ------- | ----------- |
+| `isOpen` | `bool` | false | Controls the visibility of the toast. |
+| `onClose` | `func` | | Function callback called for close requests from the close button. |
+| `appearance` | `string` | | Toasts can come in different styles that carry different semantic meanings. e.g. `"info"`, `"success"`, `"warning"`, `"error"` |
+| `actionNode` | `node` | | An optional action button. |
+| `closeNode` | `node` | `<ToastClose>` | A close button. You can set this to null to remove the close button from toasts. |
+| `children` | `node` | | Toast message content. |
+
+#### `<ToastClose>`
+
+A close button for toast messages.
+
+#### `<ToastProvider>`
+
+There can be multiple applications on the page, but all apps should share the same notification area. The first app to initialize should set up the portal and configuration; all subsequent providers will use the portal and configuration of the first to initialize.
+
+| Name | Type | Default | Description |
+| ---- | ---- | ------- | ----------- |
+| `placement` | `string` | | Where the region will exist with respect to the viewport. e.g. `"top-start"`, `"top"`, `"top-end"`, `"bottom-start"`, `"bottom"`, `"bottom-end"` |
+| `maxOpen` | `number` | 3 | The maximum number of toasts that can be open at once. |
+| `autoClose` | `number` | 5000 | The delay in milliseconds before automatically closing a toast. If set to 0, the toast must be manually dismissed. |
```

Comment:
This probably needs to be customizable at the Toast level as some Toasts might need to linger longer, or not close at all.